src/main/java/com/iridium/iridiumskyblock/listeners/PlayerInteractListener.java

```diff
@@ -36,6 +36,18 @@ public void onClick(PlayerInteractEvent event) {
             return;
         }
 
+        boolean setCancelled = true;
+        for(String world : IridiumSkyblock.getInstance().getConfiguration().whitelistedWorlds) {
+            if(player.getWorld().getName().equalsIgnoreCase(world)) {
+                setCancelled = false;
+                break;
+            }
+        }
+
+        if(setCancelled) {
+            return;
+        }
+
```

Comment:
This shouldnt be needed since we already do a check for if an island is present, which already does a World check